dg/media/coco/app/scripts/views/offline_analytics.js

```diff
@@ -0,0 +1,273 @@
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'indexeddb_backbone_config',
+    'configs',
+    'models/user_model',
+    'indexeddb-backbone',
+    'layoutmanager',
+    'libs/highstocks',
+    'offline_utils'
+    ],
+    function($, underscore, backbone, idb, all_configs, User, indexeddb, layoutmanager,highcharts, Offline){
```

Comment:
code in this file is similar to list.js file, please do check if we need all of these and please clean it up by removing all print statements